tox.ini

```diff
@@ -45,7 +45,7 @@ commands =
     make clean
     pycodestyle .
     pylint travis
```

Comment:
This is now redundant with the line below.  I think the reason you hit so many linting problems was that the line below had only been running the Python 3-related subset of pylint checks; all the other checks were only being run on this directory.

Reply:
Removed.